src/Catalyst.Core.Modules.Ledger/Ledger.cs

```diff
@@ -256,5 +292,117 @@ public void Dispose()
             Dispose(true);
             GC.SuppressFinalize(this);
         }
+
+        class ReceiptDeltaTracer : ITxTracer
+        {
+            readonly Delta _delta;
+            readonly long _deltaNumber;
+            readonly IHashProvider _hashProvider;
+            readonly Keccak _deltaHash;
+            readonly List<TransactionReceipt> _txReceipts;
+            int _currentIndex;
+
+            public ReceiptDeltaTracer(Delta delta, Cid deltaHash, long deltaNumber, IHashProvider hashProvider)
+            {
+                _delta = delta;
+                _deltaNumber = deltaNumber;
+                _hashProvider = hashProvider;
+                _deltaHash = new Keccak(deltaHash.Hash.Digest);
+                _txReceipts = new List<TransactionReceipt>(delta.PublicEntries.Count);
+            }
+
+            public IEnumerable<TransactionReceipt> Receipts => _txReceipts;
+
+            public bool IsTracingReceipt => true;
+
+            public void MarkAsSuccess(Address recipient, long gasSpent, byte[] output, LogEntry[] logs)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unused method parameter 'output'.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)